src/viewer/Viewer.jsx

```diff
@@ -143,7 +144,7 @@ export default class Viewer extends Component {
       case 'video':
         return isMobile() ? NoViewer : VideoViewer
```

Comment:
I don't know if `isMobile()` is the same as `isMobileApp()` since we only test the `window.navigator`. 

Maybe we should rename it to `isMobileBrowser()` ?

Reply:
Yeah no I think it's the same and this one should be replaced as well :)